<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1395</width>
    <height>835</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>0</y>
      <width>1411</width>
      <height>1031</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 220, 138);</string>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1321</width>
        <height>731</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/Images/Projet Redimensionner une image.jpeg);
background-repeat:no-repeat;</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>500</y>
        <width>301</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
background-image: url(:/Images/DBBBBB.jpg);
font-size:30px;
font-style:italic;
font-family:Elephant;</string>
      </property>
      <property name="text">
       <string>START</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>580</y>
        <width>171</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
background-image: url(:/Images/DBBBBB.jpg);
font-size:30px;
font-style:italic;
font-family:Elephant;
</string>
      </property>
      <property name="text">
       <string>EXIT</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>20</y>
        <width>351</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/C:/Users/ASUS/Downloads/DBBBBB.jpg);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>20</y>
        <width>351</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
background-image: url(:/Images/_8a2ccf19-ad68-4f68-95c7-734c0d82352c-removebg-preview-removebg-preview-removebg-preview.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>840</x>
        <y>60</y>
        <width>421</width>
        <height>611</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
background-image: url(:/Images/DIFFICILE.png);
background-repeat:no-repeat;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>61</y>
        <width>401</width>
        <height>611</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
background-image: url(:/Images/CUTE.png);
background-repeat:no-repeat;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>361</width>
        <height>611</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
background-image: url(:/Images/anime.jpg);
background-repeat:no-repeat;</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>570</y>
        <width>221</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
background-image: url(:/Images/COPIE.jpeg);
font-size:50px;
font-family:Chiller;
font-style:italic;
</string>
      </property>
      <property name="text">
       <string>EASY</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>570</y>
        <width>221</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
background-image: url(:/Images/COPIE.jpeg);
font-size:50px;
font-family:Chiller;
font-style:italic;
</string>
      </property>
      <property name="text">
       <string>MEDIUM</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>940</x>
        <y>570</y>
        <width>221</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/Images/COPIE.jpeg);
font-size:50px;
font-family:Chiller;
font-style:italic;
</string>
      </property>
      <property name="text">
       <string>DIFFICULT</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>10</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>1</y>
        <width>761</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 14px;
font-family: Elephant;
font-style:italic;
background-color: rgb(255, 140, 94);</string>
      </property>
      <property name="text">
       <string>ENTER YOUR NAME</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <property name="styleSheet">
      <string notr="true">background-image: url(:/Images/bois de background.jpg);
background-repeat:no-repeat;</string>
     </property>
     <widget class="QPushButton" name="pushButtonGenererPaquet">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>250</y>
        <width>131</width>
        <height>191</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>0</y>
        <width>651</width>
        <height>181</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="layoutCartes1">
       <item>
        <widget class="QPushButton" name="pushButton_9">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_8">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_7">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_6">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>470</y>
        <width>641</width>
        <height>181</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="layoutCartes2">
       <item>
        <widget class="QPushButton" name="pushButton_13">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_12">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_11">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_10">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>210</y>
        <width>541</width>
        <height>251</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="layoutCartes3"/>
     </widget>
     <widget class="QLabel" name="scoreJoueur1">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>590</y>
        <width>61</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/Images/bois2.jpg);
background-repeat:no-repeat;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="scoreJoueur2">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>30</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/Images/bois2.jpg);
backgroud-repeat:no-repeat;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>1220</x>
        <y>50</y>
        <width>101</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/Images/hi.jpg);
background-repeat:no-repeat;
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>1220</x>
        <y>510</y>
        <width>101</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/Images/cute1.jpg);
background-repeat:no-repeat;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>30</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/Images/bois2.jpg);
background-repeat:no-repeat;
font-family:ALGERIEN;
font-size:25px;
font-style:italic;</string>
      </property>
      <property name="text">
       <string>Score 1:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>590</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/Images/bois2.jpg);
background-repeat:no-repeat;
font-family:ALGERIEN;
font-size:25px;
font-style:italic;</string>
      </property>
      <property name="text">
       <string>Score 2:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <property name="styleSheet">
      <string notr="true">background-image: url(:/Images/bois2.jpg);</string>
     </property>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>80</y>
        <width>371</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size:15px;
color:black;
font-family:Elephant;
background-image: url(:/Images/DBBBBB.jpg);
font-style:italic;
background-repeat:no-repeat;</string>
      </property>
      <property name="text">
       <string>                                       le victoire </string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1395</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
